executor/distsql.go

```diff
@@ -438,6 +444,7 @@ func (e *IndexLookUpExecutor) startWorkers(ctx context.Context, initBatchSize in
 	// indexWorker will write to workCh and tableWorker will read from workCh,
 	// so fetching index and getting table data can run concurrently.
 	workCh := make(chan *lookupTableTask, 1)
+	e.collectRuntimeStatsEnabled()
```

Comment:
Prefer put `e. collectRuntimeStatsEnabled ` in `e.Open` method, it's only use for initialize.
BTW, `collectRuntimeStatsEnabled` here is strange. Look like this function only use to initialize the runtime stats.